api/v1beta1/openstackdataplanenode_types.go

```diff
@@ -102,17 +102,27 @@ func (instance *OpenStackDataPlaneNode) InitConditions() {
 	cl := condition.CreateList(
 		condition.UnknownCondition(condition.DeploymentReadyCondition, condition.InitReason, condition.InitReason),
 		condition.UnknownCondition(SetupReadyCondition, condition.InitReason, condition.InitReason),
-		condition.UnknownCondition(ConfigureNetworkReadyCondition, condition.InitReason, condition.InitReason),
```

Comment:
can we fix the kuttl tests as a result of this change?

Reply:
Ok, Looks like https://github.com/openshift/release/pull/39222 merged only on last Friday.